docs/tokens.md

```diff
@@ -66,13 +62,21 @@ TO DO
 ## Wallet
 
 TO DO
-## Issue and Transfer Tokens
+## Issuing and Transferring Tokens
 
 **TODO who are these examples meant for?**
+**TODO It looks like alice is giving carl normal tokens?**
+**TODO what is this Origin::signed business? Go to definiton takes me to the construct_runtime macro call** 
+**TODO should explain the index argument in tx.outpoint(1)**
+**TOOD: In the definition of MLT unit, it's 1000 * 100_000_000. Why written this way. One hundred billion**
+**TODO Does it cost 1000 MLT to issue a token?**
```

Comment:
It shouldn't be 1000 MLT. The figures discussed so far have been 50 MLT and 100 MLT.

Reply:
@sinitcin

Reply:
Anyhow as I understood from my chat with Sam, the 1000 MLT coins mentioned here are token Alice gives to Karl, and not a fee paid to mint the token. If I understand correctly this example does not take into account fees for minting the token.

Reply:
Cool